lib/chitter.rb

```diff
@@ -0,0 +1,44 @@
+require 'pg'
+
+class Chitter
+
+  attr_reader :peep, :username, :name, :emailaddress, :password
+  
+  def initialize(peep:, username:, name:, emailaddress:, password:)
+    @peep = peep
+    @username = username
+    @name = name
+    @emailaddress = emailaddress
+    @password = password
+  end
+
+  def self.timeline
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
I'd love to see these common if statements refactored out - you could make a private method called something like "get_connection" which handled this